src/lib/strings/index.ts

```diff
@@ -1,16 +1,9 @@
-export function prettyLineBreak(line: string): string {
-    const centerIndex = (line.length / 9) * 5;
-    const start = line.slice(0, centerIndex);
-    const end = line.slice(centerIndex).replace(/ /g, '\u00a0');
-    return start + end;
-}
-
 /**
  * Simple insecure hash function for strings
  *
  * based on https://gist.github.com/jlevy/c246006675becc446360a798e2b2d781
  *
- * @param   - str - the string to hash
```

Comment:
Webstorm warning about JSDoc not recognizing non-named params (the `-` before the `str` raises the warning)